### ISSUE TITLE
Mana Khemia 1+2 Sprite Flickering Issue

### ISSUE THREAD

**Issue by GirlsCantKissGirls** [PCSX2/pcsx2#2645]
***PCSX2 version:***
tested on stable build v1.4.0  and development build v1.5.2631-gc92e76b17

***PCSX2 options:***
All default except for speedhack enabled with default settings. I tested it with it disabled and it still persisted.

***Plugins used:***
Default native ps2 resolution with D3D 11 HW. Tested on everything else, even software.

***Description of the issue:***
Sprites in Mana Khemia 1 are flickering and "warping". I've noticed that this only happens during the daytime setting in the game. When the game goes to night or early morning it works normally but the problems happen 100% of the time during the day. Only happens when walk around the fields or school. So combat is not affected.

I have only tested Mana Khemia 1 but I have found other clips on youtube showing that Mana Khemia 2 has a similar problem.

5 Short clips showing the flickering and oddities it creates.
[mana khemia sprite clips.zip](https://github.com/PCSX2/pcsx2/files/2485892/mana.khemia.sprite.clips.zip)

Mana Khemia 2 Clip  - credit goes to TitanicGamer
https://www.youtube.com/watch?v=AmvvUv1qyNk 

***How to reproduce the issue:***
Just walk around and you will notice it. You can trigger it by standing on a certain tile and see the enemy spazz out.  The easiest place to notice it is the campus grounds and go to the incinerator to see the student standing there flickering as you walk back and forth. Wind's corridor is very easy to notice lots of flickering as well but this happens on every field.

The videos attached above show what it can do.

***Last known version to work:***
The world may never know.

***PC specifications:***
Intel i7 3770k, Nvidia GTX 1070

**Comment by arcum42** [comment]
I'm probably not the best person to actually track down the issue, but I thought I'd mention that in Mana Khemia 1, all you have to do to reproduce it is play up until the very first time you have control of the main character in the game, and watch the teacher as you walk around the classroom.

**Comment by GirlsCantKissGirls** [comment]
This issue is fixed if you enable the manual game fix GIF FIFO. Should probably add Mana Khemia 1+2 to the list of games where its required.

**Comment by lightningterror** [comment]
Can you tell me the serials for both games that you tested ?

**Comment by GirlsCantKissGirls** [comment]
I only played Mana Khemia 1 but the issue is in both of them from what I've found online.

Mana Khemia: Alchemists of Al-Revis - SLUS-21735
Mana Khemia 2: Fall of Alchemy -SLUS-21890

The issues may also persist in the Ar Tonelico games since they are running on a similar engine and were released around the same. I do not have these games to test so I can't give any feedback on them at this point.

Can confirm that Atelier Iris 1-3 ran fine without any tinkering though.
